mercury/views/dashboard.py

```diff
@@ -1,11 +1,11 @@
 from django.shortcuts import render
 from django.views.generic import TemplateView
-from ..models import SimulatedData
+from ..models import TemperatureSensor
 
 
 class DashboardView(TemplateView):
     template_name = "dashboard.html"
 
     def get(self, request, *args, **kwargs):
-        all_data = SimulatedData.objects.all().order_by("-created_at")
+        all_data = TemperatureSensor.objects.all().order_by("-created_at")
         return render(request, self.template_name, {"all_data": all_data})
```

Comment:
I suppose you just edited this because the check/test was breaking or something since you removed `SimulatedData`?